src/Mod/TechDraw/Gui/Workbench.cpp

```diff
@@ -34,14 +34,6 @@
 
 using namespace TechDrawGui;
 
-#if 0 // needed for Qt's lupdate utility

```

Comment:
Please note this block is there so that the tool bar names are marked for translation. The compiler ignores the block due to #if 0 but Qt's lupdate tool still fetches them -- if it's there.